content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -26,157 +23,289 @@ Get an overview of PodPresets at
 ```
+
+The output shows that the Pod is running:
+
 ```
 NAME      READY     STATUS    RESTARTS   AGE
 website   1/1       Running   0          4m
 ```
 
-**Pod spec after admission controller:**
-
-{{< codenew file="podpreset/merged.yaml" >}}
-
-To see above output, run the following command:
+View the Pod spec altered by the admission controller in order to see the effects of the preset
+having been applied:
 
 ```shell
 kubectl get pod website -o yaml
 ```
 
-## Pod Spec with ConfigMap Example
+{{< codenew file="podpreset/merged.yaml" >}}
 
-This is an example to show how a Pod spec is modified by the Pod Preset
-that defines a `ConfigMap` for Environment Variables.
+The `DB_PORT` environment variable, the `volumeMount` and the `podpreset.admission.kubernetes.io` annotation
+of the Pod verify that the preset has been applied.
 
-**User submitted pod spec:**
+## Pod spec with ConfigMap example
 
-{{< codenew file="podpreset/pod.yaml" >}}
+This is an example to show how a Pod spec is modified by a Pod preset
+that references a ConfigMap containing environment variables.
 
-**User submitted `ConfigMap`:**
+Here is the configuration file containing the definition of the ConfigMap:
 
 {{< codenew file="podpreset/configmap.yaml" >}}
 
-**Example Pod Preset:**
+Create the ConfigMap:
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/configmap.yaml
+```
+
+Here is the PodPreset configuration file referencing the ConfigMap:
 
 {{< codenew file="podpreset/allow-db.yaml" >}}
 
-**Pod spec after admission controller:**
+Create the preset that references the ConfigMap:
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/allow-db.yaml
+```
+
+And the configuration file containing the definition of an applicable Pod looks like this:
+
+{{< codenew file="podpreset/pod.yaml" >}}
+
+Create the Pod:
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/pod.yaml
+```
+
+View the Pod spec altered by the admission controller in order to see the effects of the preset
+having been applied:
+
+```shell
+kubectl get pod website -o yaml
+```
 
 {{< codenew file="podpreset/allow-db-merged.yaml" >}}
 
-## ReplicaSet with Pod Spec Example
+The `DB_PORT` environment variable and the `podpreset.admission.kubernetes.io` annotation of the Pod
+verify that the preset has been applied.
+
+## ReplicaSet with Pod Spec example
 
-The following example shows that only the pod spec is modified by the Pod
-Preset.
+This is an example to show that only Pod specs are modified by Pod presets. Other workload types 
+like ReplicaSets or Deployments are unaffected.
+
+Here is the PodPreset configuration file for this example:
+
+{{< codenew file="podpreset/preset.yaml" >}}
 
-**User submitted ReplicaSet:**
+Create the preset:
+
+```shell
+kubectl apply -f https://k8s.io/examples/podpreset/preset.yaml
+```
+
+Here is the configuration file containing the definition of a ReplicaSet whose deployment will result
+in the creation of three Pods:
```

Comment:
```suggestion
This manifest defines a ReplicaSet that manages three application Pods:
```
(avoiding “deployment” which could confuse a reader)

Reply:
In addition it is inline with the wording from the ReplicaSet page. Thank you for your suggestion